torch/testing/_internal/inductor_utils.py

```diff
@@ -128,3 +128,16 @@ def skip_windows_ci(name: str, file: str) -> None:
 )
 
 IS_BIG_GPU = LazyVal(lambda: HAS_CUDA and is_big_gpu())
+
```

Comment:
Here I Renamed the `clone_preserve_strides` to `clone_preserve_strides_offset` because this function also preserve the storage offset. And to distinguish from the method `clone_preserve_strides` in torch/_inductor/utils.py which does not preserve the offset.